test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj

```diff
@@ -35,10 +35,10 @@
       <NoWarn>NU1701</NoWarn>
     </PackageReference>
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="2.1.1" />
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.1.2" />
```

Comment:
I'm not entirely sure why this update was required. I was getting assembly load errors in the tests (but not from the runtime). I'm going to investigate...

Reply:
I'm pretty sure it's b/c our host is running in-proc with the tests, so the referenced version of DocDb was using this version when the script was compiled. Then when it went to run, that version wasn't found. This is why we're not directly referencing anything anymore in the host. Will follow up with @fabiocav when he's back to make sure I fully understand that it's by design. But this won't affect anything in the product as we don't reference DocDb directly anywhere.